.circleci/config.yml

```diff
@@ -589,9 +589,9 @@ workflows:
             - sanity-check
 
 # push
-      - docker-push-latest:
-          requires:
-            - packet-destroy
-          filters:
-            branches:
-              only: master
+#      - docker-push-latest:
+#          requires:
+#            - packet-destroy
+#          filters:
+#            branches:
+#              only: master
```

Comment:
I presume you are narrowing the jobs pat here to speed checking your fix and will uncomment when you have it working?   If so: super smart :)

Reply:
Yes, seems like it fails only on gke and I try to get more logs :)

Reply:
@lobkovilya  you can also try run bench tests on GKE. I think it should help to reproduce the problem.